core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -85,8 +88,8 @@ private void requestAudioFocus() {
     }
 
     private MediaPlayer createAndBindMediaPlayer(SurfaceHolder surfaceHolder, Uri videoUri) throws IOException {
-        MediaPlayer mediaPlayer = createMediaPlayer();
-        mediaPlayer.setOnPreparedListener(internalPeparedListener);
+        MediaPlayer mediaPlayer = mediaPlayerCreator.createMediaPlayer();
+        mediaPlayer.setOnPreparedListener(internalPreparedListener);
```

Comment:
@ouchadam when you mean renaming to `forwarder` you mean making this be `internalPreparedForwarder`?